stdlib/public/stubs/OptionalBridgingHelper.mm

```diff
@@ -36,12 +36,24 @@ @interface __SwiftNull : NSObject {
 }
 @end
 
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_API
+id swift_stdlib_boxUTF8CString(const char *cstr, int len);
```

Comment:
Should this be in a header file instead of declared multiple places?

Reply:
Seems reasonable